web/templates/admin/dashboard.tmpl

```diff
@@ -85,5 +85,54 @@
     </div>
     {{end}}
   </div>
+  <div class="max-w-lg">
+    <h2 class="text-xl tracking-tight font-bold text-black mt-2 mb-2">{{i18n "PrivacyModesTitle"}}</h2>
+    <p class="mb-4">
+      {{ i18n "ExplanationPrivacyModes" }} 
+      <a class="text-purple-600 font-bold" href="https://ssb-ngi-pointer.github.io/rooms2/#privacy-modes">{{ i18n "RoomsSpecification" }}</a>.
+    </p>
+  <h3 class="text-gray-600 text-sm font-bold mb-2">{{ i18n "SetPrivacyModeTitle" }}</h3>
+  <details class="mb-8 self-start w-96" id="change-privacy">
+    <summary class="px-3 py-1 w-96 rounded shadow bg-white ring-1 ring-gray-300 hover:bg-gray-100 cursor-pointer">
+    {{ i18n .CurrentMode.String }}
+    </summary>
+
+    <div class="absolute w-96 z-10 bg-white mt-2 shadow-xl ring-1 ring-gray-200 rounded divide-y flex flex-col items-stretch overflow-hidden">
+      {{ range .PrivacyModes }}
+          {{ if ne . $.CurrentMode }}
+          <form
+            action="{{urlTo "admin:dashboard:set-privacy" }}"
+            method="POST"
+            >
+            {{$.csrfField}}
+            <input type="hidden" name="privacy_mode" value="{{.}}">
+            <input
+              type="submit"
+              value="{{ i18n .String }}"
+              class="pl-3 pr-3 py-2 w-full text-left bg-white text-gray-700 hover:text-gray-900 hover:bg-gray-50 cursor-pointer"
```

Comment:
```suggestion
              class="pl-10 pr-3 py-2 w-full text-left bg-white text-gray-700 hover:text-gray-900 hover:bg-gray-50 cursor-pointer"
```

To horizontally align all the 3 texts "Open", "Community", "Restricted" regardless if they are selected or not. See line 117